profile/src/main/kotlin/com/imashnake/animite/profile/tabs/Anime.kt

```diff
@@ -29,15 +38,25 @@ fun AnimeTab(
     ) {
         // TODO: Why is this not smart-casting?
         if (!mediaCollection?.namedLists.isNullOrEmpty()) {
-            UserMediaList(mediaCollection!!.namedLists, modifier)
+            UserMediaList(
+                lists =  mediaCollection!!.namedLists,
+                modifier = modifier,
+                onNavigateToMediaItem = onNavigateToMediaItem,
+                sharedTransitionScope = sharedTransitionScope,
+                animatedVisibilityScope = animatedVisibilityScope,
+            )
         }
     }
 }
 
+@OptIn(ExperimentalSharedTransitionApi::class)
 @Composable
 private fun UserMediaList(
     lists: List<User.MediaCollection.NamedList>,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    onNavigateToMediaItem: (MediaPage) -> Unit,
+    sharedTransitionScope: SharedTransitionScope,
+    animatedVisibilityScope: AnimatedVisibilityScope,
```

Comment:
```suggestion
    onNavigateToMediaItem: (MediaPage) -> Unit,
    sharedTransitionScope: SharedTransitionScope,
    animatedVisibilityScope: AnimatedVisibilityScope,
    modifier: Modifier = Modifier
```
modifier should be [first optional param](https://github.com/slackhq/compose-lints/blob/main/docs/rules.md#modifiers-should-have-default-parameters)